res/res_usbradio.c

```diff
@@ -719,6 +719,33 @@ void ast_radio_time(time_t *second)
 	*second = ts.tv_sec;
 }
 
+/*!
+ * \brief Get serial number from device if available
+ *	This function will attempt to get the serial number from a media device
+ *
+ * \param devstr	The USB device string
+ * \param serial	Buffer to return the serial number
+ *
+ * \retval		Length of found serial number
+ */
+
+int ast_radio_get_usb_serial(char *devstr, char *serial)

```

Comment:
This function really needs to have the size of the "char *serial" buffer passed in as an argument.